docs/reference/changelog-r2021.md

```diff
@@ -13,6 +13,7 @@ Released on December XXth, 2020.
     - Added a new [`wb_supervisor_node_get_contact_point_node`](supervisor.md#wb_supervisor_node_get_contact_point_node) [Supervisor](supervisor.md) API function to get the node reference associated to a given contact point ([#2228](https://github.com/cyberbotics/webots/pull/2228)).
     - Added two new functions to the [Camera](camera.md) API called [`wb_camera_get_exposure`](camera.md#wb_camera_get_exposure) and [`wb_camera_set_exposure`](camera.md#wb_camera_set_exposure) to retrieve and change the [Camera](camera.md) exposure ([#2363](https://github.com/cyberbotics/webots/pull/2363)).
     - Added a new functionality in the [Recognition](recognition.md) node and [Camera](camera.md) API for generating segmented ground truth images ([#2199](https://github.com/cyberbotics/webots/pull/2199)).
+    - **Added a new [`wb_inertial_unit_get_quaternion`](inertialunit.md#wb_inertial_unit_get_quaternion) [InertialUnit](inertialunit.md) API function to get the orientation measurement represented as a quaternion ([#2424](https://github.com/cyberbotics/webots/pull/2424)).**
```

Comment:
Why this entry is written in bold?
Does it break compatibility with previous version? if not it should not be in bold to be consistent with the rest of the ChangeLog.

Reply:
Yes, and we should also mention here in bold that a function was removed and another one was added (lookup table and noise).

Reply:
It seems this comment has still not been solved, and in addition, the removal of the ``lookupTable`` field should be mentioned too.